### ISSUE TITLE
Add refresh/reload button for sections, functions, etc..

### ISSUE THREAD

**Issue by wargio** [hteso/iaito#126]
I would love to have a refresh/reload button for functions and sections since when i type a cmd like `af @ addr` the new function is not being added, and the same for sections and any other tab (like these two)

**Comment by Maijin** [comment]
#103 related

**Comment by radare** [comment]
Refresh buttons are an antipattern but definitively necessary with the current design of iaito. What about a keybinding to not make the ui more bloated?

**Comment by wargio** [comment]
I agree, but i would suggest to still add it as a button, under some "hidden" menu, because it has to be like that (for a GUI).

**Comment by ballessay** [comment]
Wouldn't it be the easiest solution to update the views after a command finished?

**Comment by wargio** [comment]
no, if you have a huge binary (like me), it will freeze the GUI

**Comment by ballessay** [comment]
@wargio makes sense. Maybe it would be better to only refresh the visible widgets when a shortcut is triggerd? Or every widget gets its own refresh short cut.

**Comment by Jo-Con-El** [comment]
I'd be inclined to a) ask the user after making those changes and before going back to the main interface if they want a refresh, possibly in a way that is persistent over the current session (or ever better, per binary or a global setting to make it positive by e.g. file size), and b) if the user chooses to go without refreshing, use a visual hint to tell them that a reload is pending/needed to get the latest changes.  It can be from an asterisk in the title bar after the file name (too subtle for me) or an entire message " (manual reload needed)".

**Comment by wargio** [comment]
agree

**Comment by ballessay** [comment]
For a) && b) it would be nice to know after which commands a refresh is needed or should be assumed that every console command triggers the nag-dialog/window modified hint?

c) Is/Should be implemented with #135
